README.md

```diff
@@ -41,6 +41,117 @@ the last few lessons.
 
 Put your code in `index.js`.
 
+While you will want to be guided by the tests, you will implement the following
+functions. To make the tests easier to read, we've provided the _signatures_ of
+the functions.
+
+A function _signature_ is the function name, the arguments it expects, and what
+the function returns.
+
+### `createEmployeeRecord`
+
+* **Argument(s)**
+  * 4-element Array of a `String`, `String`, `String`, and `Number`
+    corresponding to a first name, family name, title, and pay rate per hour
+* **Returns**
+  * JavaScript `Object` with keys:
+    * `firstName`
+    * `familyName`
+    * `title`
+    * `payPerHour`
+    * `timeInEvents`
+    * `timeOutEvents`
+* **Behavior**
+  * Loads `Array` elements into `Object` and initializes Arrays for storing
+    time in events and time out events
+
+### `createEmployees`
+
+* **Argument(s)**
+  * `Array` of `Arrays`
+* **Returns**
+  * `Array` of `Object`s
+* **Behavior**
+  * Converts each nested `Array` into an employee record using
+    `createEmployeeRecord` and accumulates it to a new `Array`
+
+### createTimeInEvent
+
+* **Argument(s)**
+  * A date stamp
+* **Returns**
+  * The record used as execution context
+* **Behavior**
+  * Add an `Object` with keys:
+    * `type`: Set to `"TimeIn"`
+    * `hour`: Derived from the argument
+    * `date`: Derived from the argument
+
+
+### createTimeOutEvent
+
+* **Argument(s)**
+  * A date stamp
+* **Returns**
+  * The record used as execution context
+* **Behavior**
+  * Add an `Object` with keys:
+    * `type`: Set to `"TimeOut"`
+    * `hour`: Derived from the argument
+    * `date`: Derived from the argument
+
+### hoursWorkedOnDate
+
+* **Argument(s)**
+  * A date of the form `"YYYY-MM-DD"`
+* **Returns**
+  * Minutes worked
+* **Behavior**
+  * Given a date, find the number of hours elapsed between that date's
+    timeInEvent and timeOutEvent
+
+### wagesEarnedOnDate
+
+* **Argument(s)**
+  * A date of the form `"YYYY-MM-DD"`
+* **Returns**
+  * Pay owed
+* **Behavior**
+  * Using `hoursWorkedOnDate`, multiply the hours by the record's
+    payRate to determine amount owed. Amount should be returned as a number.
+
+### allWagesFor
+
+* **Argument(s)**
+  * _None_
+* **Returns**
+  * Pay owed for all dates
+* **Behavior**
+  * Using `wagesEarnedOnDate`, accumulate the value of all dates worked by the
+    employee in the record used as context. Amount should be returned as a
+    number.
+
+### findEmployeebyFirstName
```

Comment:
Good candidate for a function to move to the first spot in this list - it doesn't depend on anything and the input and output are clear